components/intl.rst

```diff
@@ -179,6 +179,29 @@ You may convert codes between two-letter alpha2 and three-letter alpha3 codes::
 
     $alpha2Code = Countries::getAlpha2Code($alpha3Code);
 
+Numeric Country Codes
+~~~~~~~~~~~~~~~~~~~~~
+
+The :class:`Symfony\\Component\\Intl\\Countries` class also provides access to the
+numeric country codes according to the `ISO 3166-1 numeric`_ list::
+
+    use Symfony\Component\Intl\Countries;
+
+    \Locale::setDefault('en');
+
+    $numericCodes = Countries::getNumericCodes();
+    // ('alpha2Code' => 'numericCode')
+    // => ['AA' => '958', 'AD' => '020', ...]
+
+    $numericCode = Countries::getNumericCode('FR');
+    // => '250'
+
+    $alpha2 = Countries::getAlpha2FromNumeric('250');
+    // => 'FR'
+
+    $exists = Countries::numericCodeExists('250');
+    // => true
+
```

Comment:
Since this is a new feature, we must add a note like the following:

```rst
.. versionadded:: 6.4

    The support for numeric country codes was introduced in Symfony 6.4.
```

Reply:
Done